options.go

```diff
@@ -19,8 +19,36 @@ func withDefaultCheckListener() Option {
 	}
 }
 
+// WithClassification allows you to create Health object for specific usage (liveness/readiness)
+func WithClassification(classification string) Option {
```

Comment:
I wonder if it would make sense to wait until the metrics responsibility is extracted from the health checker to a metrics interface or a CheckListener+HealthListener combo. Then the healthchecker itself could be kept free of this classification thing.

Reply:
Please review my comment https://github.com/AppsFlyer/go-sundheit/issues/24#issuecomment-767454467 as it will eventually remove the use of classification for health service.

Reply:
Yeah, that (your comment on the other thread) was my understanding as well. My question here is whether it makes sense to introduce classification on the health service level if the plan is to deprecate it and move it to the metrics level anyway. It feels like an unnecessary breaking change to me.